### PR TITLE
Add org.codeberg.Qwel.egui_picross

### PR DESCRIPTION
**Pull request by 0Qwel**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** [I am upstream, not sure what to link](https://codeberg.org/Qwel)
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: page.codeberg.Qwel.egui_picross.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+modules:
+  - name: page.codeberg.Qwel.egui_picross
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      build-args:
+        - --share=network
```
Comment (hfiguiere) [inline page.codeberg.Qwel.egui_picross.yml:18]: build must be offline. this is not allow and will fail.

File: page.codeberg.Qwel.egui_picross.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+app-id: page.codeberg.Qwel.egui_picross
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: page.codeberg.Qwel.egui_picross
+finish-args:
+  - --socket=wayland
```
Comment (hfiguiere) [inline page.codeberg.Qwel.egui_picross.yml:9]: `--share=ipc` is necessary for X11 (even fallback)

File: page.codeberg.Qwel.egui_picross.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+app-id: page.codeberg.Qwel.egui_picross
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```
Comment (hfiguiere) [inline page.codeberg.Qwel.egui_picross.yml:3]: current runtime is 23.08

**Comment by 0Qwel** [comment]
Requested changes applied

File: page.codeberg.Qwel.egui_picross.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        - CARGO_HOME: /run/build/egui_picross/cargo
+        - LEVELS_PATH: ~/.var/app/page.codeberg.Qwel.egui_picross/data/
+    sources:
+      - cargo-sources.json
+      - type: file
+        path: config.toml
```
Comment (hfiguiere) [inline page.codeberg.Qwel.egui_picross.yml:24]: this is included in the cargo-sources.json.

File: page.codeberg.Qwel.egui_picross.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: page.codeberg.Qwel.egui_picross
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        - CARGO_HOME: /run/build/egui_picross/cargo
```
Comment (hfiguiere) [inline page.codeberg.Qwel.egui_picross.yml:19]: this won't work since the module name is not `egui_picross`

**Pull request closed without merging**